hb_report/hb_report.in

```diff
@@ -164,9 +164,9 @@ def parse_argument(argv):
         if args == "-e":
             constants.EDITOR = option
         if args == "-p":
-            constants.SANITIZE += " %s" % option
+            constants.SANITIZE_RULE += " %s" % option
```

Comment:
Changing the value of `constants` is rare. Why are you calling `constant` if it has a variable content?